tests/Doctrine/Tests/ORM/Functional/Ticket/GH2947Test.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+
+use Doctrine\Tests\OrmFunctionalTestCase;
+
+/**
+ * @group 2947
+ */
+class GH2947Test extends OrmFunctionalTestCase
```

Comment:
[`Doctrine\Tests\ORM\Query\QueryTest#testResultCacheEviction()`](https://github.com/doctrine/doctrine2/pull/6417/files#diff-8839a19e4228e39f37c3000d51e43beeR250) not strictly unit test though but should be enough 😉